packages/ia-topnav/src/media-button.js

```diff
@@ -0,0 +1,100 @@
+import { html } from 'lit-element';
+import TrackedElement from './tracked-element';
+import icons from './assets/img/icons';
+import toSentenceCase from './lib/toSentenceCase';
+import mediaButtonCSS from './styles/media-button';
+
+class MediaButton extends TrackedElement {
+  static get styles() {
+    return mediaButtonCSS;
+  }
+
+  static get properties() {
+    return {
+      config: { type: Object },
+      icon: { type: String },
+      href: { type: String },
+      label: { type: String },
+      mediatype: { type: String },
+      openMenu: { type: String },
+      selected: { type: Boolean },
+      followable: { type: Boolean },
```

Comment:
What does this `followable` property do?

Reply:
Whether the click/tap default is prevented. Allows for the donate link (and previously the upload link)